src/js/models/v2/state/CoreRecipient.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * CoreRecipient.js
+ * Created by Lizzie Salita 5/1/18
+ */
+
+import { formatNumberWithPrecision, calculateUnitForSingleValue, formatMoneyWithPrecision } from 'helpers/moneyFormatter';
+
+const CoreRecipient = {
```

Comment:
What other pages/features will this model be shared with? If it is one-time use/only for state profile metadata, it may make more sense to include it as a `Base` object that directly populates from the API (make just 1 API call, but pass the result into `populate` methods for two separate models).

Reply:
Removed the `CoreRecipient` file; moved its properties to `BaseStateProfile`